libraries/door43_tools/project_printer.py

```diff
@@ -70,11 +74,14 @@ def print_project(self, project_id):
             html = App.cdn_s3_handler().get_file_contents(print_all_key)
         return html
 
-    def frontToBack(self, file):
-        # make front and back sort before and after numeric files
-        if(file.find('front') >= 0):
+    def frontToBack(self, file_path):
```

Comment:
@bspidel This was improperly ordering NON OBS projects, such as the Bible, putting them all out of order. frontToBack gets called for ALL projects/resources, not just OBS. Made this fix to hopefully handle anything with obs in the project ID.